tests/tests_api/test_teacher.py

```diff
@@ -232,6 +233,7 @@ def test_dashboard_teacher_list(api_client, faker, availability_slots):
     teacher_data = DashboardTeacherSerializer(teacher).data
     teacher_data["utc_timedelta"] = f"UTC{sign}{utc_offset_hours:02}:{utc_offset_minutes:02}"
     assert_response_data_list(response.data, [teacher_data])
+    teacher.has_prior_teaching_experience == response.data[0]["has_prior_teaching_experience"]
```

Comment:
Note, I have just added it like that for now, but in the tests issue we agreed that it will be refactored in the future to follow same pattern as everythere